### PR TITLE
Fix issue with replica count display

### PR DESCRIPTION
**Pull request by lucferbux**
## Description

Closes: https://issues.redhat.com/browse/RHOAIENG-2269

## How Has This Been Tested?

1. Create a New KServe Model
2. Edit it and change the replica count
3. Check that the replica count has changed in the expanded board
4. Check that the edit panel has reflected the changes

Screenshot 2024-02-09 at 13 05 37
Screenshot 2024-02-09 at 13 02 39

## Request review criteria:

Self checklist (all need to be checked):
- [x] The developer has manually tested the changes and verified that the changes work
- [x] Commits have been squashed into descriptive, self-contained units of work (e.g. 'WIP' and 'Implements feedback' style messages have been removed)
- [x] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [x] The developer has added tests or explained why testing cannot be added (unit tests & storybook for related changes)

If you have UI changes: 

- [ ] Included any necessary screenshots or gifs if it was a UI change.
- [ ] Included tags to the UX team if it was a UI/UX change (find relevant UX in the [SMEs](https://github.com/opendatahub-io/odh-dashboard/tree/main/docs/smes.md) section).

After the PR is posted & before it merges:
- [ ] The developer has tested their solution on a cluster by using the image produced by the PR to `main`

### CODE REVIEW

File: frontend/src/__tests__/cypress/cypress/e2e/modelServing/ServingRuntimeList.cy.ts [hunk 1]
```diff
@@ -963,4 +963,37 @@ describe('Serving Runtime List', () => {
         .should('exist');
     });
   });
+
+  it.only('Check number of replicas of model', () => {
```
Comment (dpanshug) [inline frontend/src/__tests__/cypress/cypress/e2e/modelServing/ServingRuntimeList.cy.ts:967]: Remove `.only`

Comment (lucferbux) [reply frontend/src/__tests__/cypress/cypress/e2e/modelServing/ServingRuntimeList.cy.ts:967]: wow thanks

**Review by dpanshug — APPROVED** [review]
/lgtm

**Pull request merged**